src/react-apps/applications/shared/src/file-editor/FileEditor.tsx

```diff
@@ -136,33 +136,6 @@ class FileEditor extends React.Component<IFileEditorProvidedProps, IFileEditorSt
       });
   }
 
-  public getFolderText(): string {
```

Comment:
The gets in this file are returning net::ERR_ABORTED 500 (Internal Server Error) in the beginning
And are followed by "WARNING: Promise with no error callback:2"

Reply:
@malmil This is not related to the files that are loaded _into_ the editors, but the different modules that are loaded in dev build to get the editor workers running. This is only a problem when running with local dev, and can be resolved locally by commenting in the last 3 lines in Views/ServiceDevelopment/index.cshtml.

Reply:
Changed the imports in Views/ServiceDevelopment/index.cshtml to happen only when in local dev.